src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Providers/GetAzureProviderCmdlet.cs

```diff
@@ -89,5 +91,38 @@ public override void ExecuteCmdlet()
                 this.WriteObject(providers, enumerateCollection: true);
             }
         }
+
+        private PSResourceProvider[] ListPSResourceProviders()
```

Comment:
I assumed PS stands for PowerShell, its good to call it ListPowerShellResourceProviders